Fix 'documentation' github actions

Issue overview
--------------

The documentation github actions build fails due to an apt error.

### Details

Example of a failing run https://github.com/NREL/EnergyPlus/pull/8401/checks?check_run_id=1486213415, and relevant info from the log below:
```
Get:29 http://azure.archive.ubuntu.com/ubuntu bionic/universe amd64 texlive-xetex all 2017.20180305-1 [10.7 MB]
Fetched 208 MB in 6s (37.3 MB/s)
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/poppler/libpoppler73_0.62.0-2ubuntu2.10_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.

```

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

--- a/.github/workflows/documentation.yml
+++ b/.github/workflows/documentation.yml
@@ -16,7 +16,7 @@ jobs:
         python-version: 3.7
 
     - name: Set up LaTeX
-      run: sudo apt-get install texlive texlive-xetex texlive-science
+      run: sudo apt update && sudo apt install -y texlive texlive-xetex texlive-science
 
     - name: Create Build Environment
       run: cmake -E make_directory ${{runner.workspace}}/EnergyPlus/doc/build

